File: noti-service/src/main/java/com/ezbuy/notiservice/repository/query/NotificationCategoryQuery.java
package com.ezbuy.notiservice.repository.query;

public interface NotificationCategoryQuery {
    String FIND_CATEGORY_ID_BY_TYPE = """
                SELECT 
                  nc.id 
                FROM 
                  notification_category nc 
                WHERE 
                  nc.type = : categoryType 
                  AND status = 1
            """;
}

File: notification-service/src/main/kotlin/com/ezbuy/notificationservice/repository/query/NotificationCategoryQuery.kt
package com.ezbuy.notificationservice.repository.query

interface NotificationCategoryQuery {
    companion object {
        const val FIND_CATEGORY_ID_BY_TYPE = """
            SELECT 
              nc.id 
            FROM 
              notification_category nc 
            WHERE 
              nc.type = : categoryType 
              AND status = 1
        """
    }
}
